File: quicksort.py
def quicksort(arr):
    if len(arr) <=1:
        return arr
    pivot = arr[0]
    left=[]
    right=[]
    for ind in range(1,len(arr)):
        if arr[ind]<=pivot:
            left.append(arr[ind])
        else:
            right.append(arr[ind])
    leftsort=quicksort(arr)
    rightsort=quicksort(arr)
    return leftsort+[pivot]+rightsort

File: quicksort.js
function quicksort(arr){
    if(arr.length<=1) return arr
    const pivot = arr[0]
    const left = []
    const right = []
    for (let i=1;i<arr.length; i++){
        let val = arr[i]
        if(val<=pivot) left.push(val)
        else right.push(val)
    }
    const leftsort = quicksort(arr)
    const rightsort=quicksort(arr)
    return [...leftsort, pivot, ...rightsort]
}